scripts/functions/extended_crafting/compression.zs

```diff
@@ -0,0 +1,28 @@
+/*
+	Script made by Zombie_Konsti
+*/
+
+#modloaded extendedcrafting
+#priority 3400
+import crafttweaker.item.IItemStack;
+import mods.extendedcrafting.CompressionCrafting;
+
+
+function add(data as int[IItemStack][IItemStack][IItemStack][[int]]) {
+	for power, items in data {
+		for catalyst, recipe in items {
+			for	output, inputs in recipe {
+				for item, amount in inputs {
+				if(power.length == 1) { CompressionCrafting.addRecipe(output, item, amount, catalyst, power[0]); }
+				else if(power.length == 2) { CompressionCrafting.addRecipe(output, item, amount, catalyst, power[0], power[1]); }
+				}
```

Comment:
You should indent the `if` and `else if` one stage further, as you also have done in the other files of your PR.